.github/workflows/pre-commit_hooks.yaml

```
@@ -11,10 +11,10 @@ jobs:
     - name: checkout
       uses: actions/checkout@v2
 
-    - name: set up python 3.9
+    - name: set up python 3.12
       uses: actions/setup-python@v1
```

actionlint (1.7.10)

[error] the runner of "actions/setup-python@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)